.travis.yml

```diff
@@ -0,0 +1,5 @@
+language: python
+python:
+  - "2.7"
+install: pip install -r requirements.txt
+script: DJANGO_SETTINGS_MODULE="settings" nosetests --with-coverage --with-django
```

Comment:
Sweet! - we need a few things added in here:
Calling pep8 checks
Calling pylint checks
After running tests, we need to run the coverage report.

Once you've got that running we may need to increase the coverage if we are missing a lot ;-)